app/components/extras.py

```diff
@@ -0,0 +1,45 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Any, final
+
+import discord as dc
+from discord.ext import commands
+
+from app.utils import is_dm, is_mod, try_dm
+
+if TYPE_CHECKING:
+    from app.bot import GhosttyBot
+
+
+@final
+class Extras(commands.Cog):
+    def __init__(self, bot: GhosttyBot) -> None:
+        self.bot = bot
+
+    @commands.command(name="sync", description="Sync command tree.")
+    @commands.guild_only()
+    async def sync(self, ctx: commands.Context[Any]) -> None:
+        assert not is_dm(ctx.author)
+        if not is_mod(ctx.author):
+            return
+
+        await self.bot.tree.sync()
+        await try_dm(ctx.author, "Command tree synced.")
+
+    @dc.app_commands.command(
+        name="status", description="Export Ghostty Bot Status Message"
+    )
+    async def ghostty_bot_status(self, interaction: dc.Interaction) -> None:
+        user = interaction.user
+        member = self.bot.ghostty_guild.get_member(user.id)
+        if member is None or not is_mod(member):
+            await interaction.response.send_message(
+                "Sorry, only mods can use this command", ephemeral=True
```

Comment:
```suggestion
                "Only mods can use this command.", ephemeral=True
```

I don't think any other “only mods can use this” has a sorry ^^. (Also added a full stop.)